config/queue.php

```diff
@@ -39,37 +38,32 @@
             'driver' => 'database',
             'table' => 'jobs',
             'queue' => 'default',
-            'expire' => 60,
+            'retry_after' => 90,
         ],
 
         'beanstalkd' => [
             'driver' => 'beanstalkd',
             'host' => 'localhost',
             'queue' => 'default',
-            'ttr' => 60,
```

Comment:
Is this option still looked at in the code?

Reply:
@LukeTowers `ttr` was renamed to `retry_after` like the database change above.